app/assets/stylesheets/modules/subnavbar.scss

```diff
@@ -25,7 +25,15 @@
     color: $white;
   }
 
+  .sul-services-menu {
+    min-width: 300px;
```

Comment:
Could this be done with padding instead? Is 300px going to be correct for all zoom levels and screen sizes?

Reply:
@jcoyne I discovered `white-space: nowrap` and gave that a try. At smaller screen sizes / higher zoom levels, the menu dropdown is 100% wide and `white-space` is reset to `normal`. What do you think?